akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -371,7 +371,14 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
     case other             ⇒ super.unhandled(other)
   }
 
-  def initJoin(): Unit = sender() ! InitJoinAck(selfAddress)
+  def initJoin(): Unit = {
```

Comment:
yeah, this might have been the original reason for the idea, but it makes sense to me in all cases